src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+    private int numberOfBall = random.nextInt(99);
```

Comment:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants